src/commands/create_keys.py

```diff
@@ -46,6 +46,12 @@
     type=str,
     callback=validate_eth_address,
 )
+@click.option(
+    '--withdrawal-address',
+    help='The address of the vaultk.',
```

Comment:
It's a bit confusing that we have both --vault and --withdrawal-address with the same description. Maybe we should rename it to `--eigenpod-address`?